### ISSUE TITLE
Qt GUI changes

### ISSUE THREAD

**Issue by dpk9** [Wyss/automated-chromatography#10]
* Tab order should be reviewed
* Button names could be clearer
* Check (and other icon) paths should be relatively defined, not absolute

**Comment by dpk9** [comment]
see f3c60e3e3397f28e914526a0f14c9ef0bb6801ae

**Comment by dpk9** [comment]
dispense checkboxes are smashed together

**Comment by dpk9** [comment]
too broad
